src/components/Applikation.vue

```diff
@@ -1,179 +1,131 @@
-      som skal lytte på <strong>hashchange</strong> event, og efterfølgende implementere logikken når # ændres.
-    </div>
-    <h4>Du er på trin {{ step }}/{{ maxStep }}</h4>
-    <form class="my-5">
-      <div v-show="step === 1" class="form-group">
-        <label class="form-label" for="navn"> Navn </label>
-        <input id="navn" class="form-input" required value="Jens" name="navn" type="text" />
-      </div>
-      <div v-show="step === 2" class="form-group">
-        <label class="form-label" for="efternavn"> Efternavn </label>
-        <input id="efternavn" class="form-input" required value="Hansen" name="efternavn" type="text" />
-      </div>
-    </form>
-    <div v-show="step === maxStep">
-      <div class="alert alert-info my-5">
-        <div class="alert-body">
-          <p class="alert-heading">Sidste trin</p>
-          <p class="alert-text">Du har nået sidste trin i guiden ...</p>
-        </div>
-      </div>
-    </div>
-    <div>
-      <button class="button button-primary" @click="decreaseStep()">Forrige</button>
-      <button class="button button-primary" @click="increaseStep()">Næste</button>
-    </div>
-    <h3>Parameter variant:</h3>
-    <div>
-      Eksempel på brug af variant parameter, som udenfor sandkassen specificeres som Vue property fra Virksomhedsguiden. I denne sandkasse angives
-      varianten i
-      <strong>src/App.vue</strong> for at have en effekt.
-    </div>
-    <pre>{{ variant }}</pre>
-    <div class="py-5 align-text-center" :style="{ 'background-color': variantColor }">
-      Baggrundsfarven bestemmes af parameter varianten ({{ variantName }})
-    </div>
-    <h3>API:</h3>
-    <div>
-      Eksempel på brug af <a href="https://www.npmjs.com/package/axios">Axios</a> biblioteket til at kalde eksternt API og vise spinner ved asynkrone
-      operationer
-    </div>
-    <div>
-      <div v-if="pending" class="spinner" aria-label="Henter indhold" />
-      <div v-if="error" class="alert alert-error my-5" role="alert" aria-atomic="true">
-        <div class="alert-body">
-          <p class="alert-heading">Fejl</p>
-          <p class="alert-text">API request failed</p>
-        </div>
-      </div>
-      <pre v-else class="my-5">{{ response }}</pre>
-      <button class="button button-primary" @click="callAPI()">API kald</button>
-      <button class="button button-primary" @click="callAPI(true)">API kald med fejl</button>
+    <h1>Leverandør-applikation Sandkasse Miljø</h1>
+    <div class="my-5">
+      Denne applikation kan bruges som skabelon til udvikling af leverandør-applikation, der kan integreres i
+      <a href="https://www.virksomhedsguiden.dk" target="_blank">Virksomhedsguiden</a> som et Vue komponent. Siden indholder desuden teknisk
+      information om forskellige aspekter af en leverandør-applikation. Se <strong>README.md</strong> for instruktioner. Der henvises desuden til den
+      tekniske og design vejledning, som er blevet udleveret for yderligere information.
```

Comment:
den tekniske vejledning og design-vejledningen, som er blevet udleveret, for yderligere information.